qa-pipelines/tasks/delete-aks-resource-group.yml

```diff
@@ -0,0 +1,22 @@
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: splatform/cf-ci-orchestration
+    tag: latest
+inputs:
+- name: ci
+- name: pool.kube-hosts
+
+params:
+  KUBE_REGISTRY_HOSTNAME: ""
+  KUBE_REGISTRY_USERNAME: ""
+  KUBE_REGISTRY_PASSWORD: ""
+  KUBE_ORGANIZATION: ""
+  HA: false # true, false, or "scaled"
```

Comment:
annotations aren't needed here. Also, is this param even being used by anything in the delete-terraform task?

Reply:
I just copied this from cf-teardown.yml. As the requirement of such annotations were triggered when using new terrform which I am not using.
Please let me know which line to delete here.